InGreedIoApi/Program.cs

```diff
@@ -158,11 +158,8 @@
     var services = scope.ServiceProvider;
 
     var context = services.GetRequiredService<ApiDbContext>();
-    if (context.Database.GetPendingMigrations().Any())
-    {
-        context.Database.EnsureDeleted();
-        context.Database.EnsureCreated();
-    }
+    context.Database.Migrate();

```

Comment:
Are you sure there shouldn't be this if `context.Database.GetPendingMigrations().Any()`?

Reply:
this doesnt work i dont know why

Reply:
maybe leave ensuredeleted created

Reply:
Can you try:
```csharp
    if(context.Database.IsNpgsql())
    {
      if (context.Database.GetPendingMigrations().Any())
      {
          context.Database.Migrate();
      }
    }
    else
    {
      context.Database.EnsureCreated();
    }
```
?
It is code from our dotnet project from last term